.cloud/.azure/deploy.params.json

```diff
@@ -1,7 +1,4 @@
 {
```

Comment:
I will suggest not to change the basic format of the params file. 
but instead inside the code, read only the parameters section of the file. 

reason is we want to give user a very cli native experiences.

so inside the code just read the parameters part.

Reply:
reverted the change and handling this inside code